core/shared/src/test/scala/zio/prelude/CoherentSpec.scala

```diff
@@ -18,21 +19,38 @@ object CoherentSpec extends DefaultRunnableSpec {
         val instance = implicitly[EqualIdentity[String]]
```

Comment:
it would be more elegant to just use the newly added extension methods here (and in the other tests